packages/model-viewer/scripts/ci-run-tests.sh

```diff
@@ -20,11 +20,8 @@ set -x
 
 if [ "${TEST_TYPE}" = "unit" ]; then
 
-  npm run test
+  npm run wct-local-chrome
```

Comment:
Still confused by this name; are we really still using WCT?

Reply:
Oh wait, we don't actually need this file anymore, right? Since we're not using Travis anymore?

Reply:
Good catch, this script is vestigial and has been removed 👍